docs/setup.md

```diff
@@ -99,21 +92,55 @@ pip3 install --user sel4-deps
 
 ### Haskell Stack
 
-After installing
-[haskell-stack](https://docs.haskellstack.org/en/stable/), make sure
-you've adjusted your `PATH` to include `$HOME/.local/bin`, and that you're
-running an up-to-date version:
+After installing [haskell-stack](https://docs.haskellstack.org/en/stable/)
+(already included in the packages above on Mac and Ubuntu), make sure you've
+adjusted your `PATH` to include `$HOME/.local/bin`, and that you're running an
+up-to-date version:
 
 ```bash
 stack upgrade --binary-only
 which stack # should be $HOME/.local/bin/stack
 ```
 
+## Checking out the repository collection
+
+The seL4 repositories use the [Google `repo` tool][repo] for configuration
+control and managing sets of repositories. For verification, this means in
+particular managing the correct combinations of the proofs, the kernel sources,
+and the Isabelle/HOL theorem prover.
+
+The [verification-manifest] repository records which versions of these are know
+to work well together.
+
+To check out a consistent set of repositories, run the following steps:
+
+```sh
+mkdir verification
+cd verification
+repo init -u https://git@github.com/seL4/verification-manifest.git
+repo sync
+```
+
+If you are developing proofs, intending to contribute, and have `ssh` set up
+for GitHub, use
+
+```sh
+repo init -m devel.xml -u ssh://git@github.com/seL4/verification-manifest.git
+```
+
+instead for the `init` line. The `-m devel.xml` gives you the `master` branch of
+the `l4v` repository instead of the last known-good version. To set up `git` for
+`ssh` make sure to use the `ssh://` protocol explicitly as above instead of just
+`git@github.com`, because the short form seems to confuse `repo`.
+
+[repo]: https://gerrit.googlesource.com/git-repo/+/HEAD/README.md
+[verification-manifest]: https://github.com/seL4/verification-manifest
+
 ## Isabelle Setup
```

Comment:
I notice that we are now building `HOL` instead of `HOL-Word` as part of these instructions. I believe this is only for newer versions of Isabelle, right? At some point the version we were using in timeprot was not up to date, so I had to remember the difference (in fact in my head there was a third way of building Word at some point).

Anyway, if you think this is an issue others might run into, we might want to document that here. If that's not likely, then no change is needed.

Reply:
It should be always safe to just build HOL. It'll just take a bit longer to build older sessions the first time because they also need to build HOL-Word, but that will happen automatically.